--- librsync-0.9.5/Makefile.am.org	Fri Sep 14 03:35:04 2001
+++ librsync-0.9.5/Makefile.am	Mon Mar  4 00:41:30 2002
@@ -17,12 +17,6 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
-
-			      # Thus spake the master programmer:
-			      # 
-			      # ``When the program is being tested, it
-			      # is too late to make design changes.''
-
 SUBDIRS = popt
 
 lib_LTLIBRARIES = librsync.la
diff -uNr librsync-0.9.5.1/configure.in librsync-0.9.5.1.orig/configure.in
--- librsync-0.9.5.1/configure.in	Sun Jul 21 00:31:41 2002
+++ librsync-0.9.5.1.orig/configure.in	Sat Jun 29 01:46:13 2002
@@ -157,7 +157,7 @@
 AC_SUBST(CC_SHOBJ_FLAG)
 AC_SUBST(CCMALLOC_LIB)
 AC_SUBST(net_libs)
+AC_LIBOBJ
-AC_SUBST(LIBOBJS)
 AC_SUBST(LIBTOOL_DEPS)
 AC_SUBST(BUILD_POPT)
 
