diff -Nur librsync-0.9.6.org/configure.ac librsync-0.9.6/configure.ac
--- librsync-0.9.6.org/configure.ac	Sun Aug  3 14:42:40 2003
+++ librsync-0.9.6/configure.ac	Sat Aug  9 11:17:22 2003
@@ -6,7 +6,7 @@
 AC_REVISION([$Revision$])
 AC_COPYRIGHT([Copyright (C) 1999, 2000, 2001, 2002 by Martin Pool <mbp@samba.org>])
 AC_CONFIG_SRCDIR([trace.c])
-AC_CONFIG_HEADER([config.h])
+AM_CONFIG_HEADER([config.h])
 AM_INIT_AUTOMAKE
 
 # GNU library versioning: This is NOT the librsync release number.
@@ -18,7 +18,7 @@
 AC_DISABLE_SHARED
 
 # Checks for programs.
-AC_GNU_SOURCE
+#AC_GNU_SOURCE
 AC_PROG_CC
 AC_ISC_POSIX
 AC_PROG_CPP
